docker/docker-compose-production.yml

```diff
@@ -30,8 +30,9 @@ services:
     networks:
       - stb-backend
       - stb-frontend
-    command: gunicorn stroyprombeton.wsgi:application -c /etc/gunicorn.py -b 0.0.0.0:$VIRTUAL_HOST_PORT
+    command: gunicorn stroyprombeton.wsgi:application -c /etc/gunicorn.py -b 0.0.0.0:8000
 
+  # @todo #232:60m Create separated DB for stage
```

Comment:
Should we either create a separate database in existenting postgres instance or new one instance of Postgesql? What exactly have you meant?

Reply:
@artemiy312 , it's realization. Task implementor should think about it.
Requirements are to have separated DB. Stage should have it's own data independent of prod

Reply:
clarified for a little task requirements